starfish/image/_stack.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, image_partition):
             self._num_zlayers = image_partition.get_dimension_shape(Indices.Z)
         else:
             self._num_zlayers = 1
-        self._tile_shape = tuple(image_partition.default_tile_shape)
+        self._tile_shape = image_partition.default_tile_shape
```

Comment:
What type is an image_partition?

Reply:
It's a TileSet.

Reply:
Can you point me to the tileset object definition? (Just for my own learning, I don't have any concerns I want to address with the code, feel free to merge)

Reply:
@csweaver It lives here: https://github.com/spacetx/slicedimage/blob/master/slicedimage/_tileset.py